meetups/2020/1/25/talks/anchal.txt

```diff
@@ -0,0 +1,15 @@
+1. Introduction
+2. How to make build process easy
+    2.1 Introduction to containerization i.e docker
+    2.2 What and Why of Docker?
+    2.3 What is Docker Hub?
+    2.4 Talk and demo of Docker run command   
+    2.5 Dockerfile and docker build (Building apache web server)
+    2.6 Ports in Docker
+    2.6 How to stop a container
+3  Deploying  our code
+    3.1 What is AWS?
+    3.2 How to run an Ec2 instance?
+    3.3 What is Amazon S3?
+    3.4 Hosting a static website in S3 bucket\
```

Comment:
Is this required for the beanstalk part? It seems a little off theme. Please correct me if I'm wrong. :smile: